src/commands/gate/renderer.ts

```diff
@@ -59,6 +59,10 @@ export const renderRuleEvaluation = (ruleEvaluation: RuleEvaluation): string =>
   }
 
   fullStr += `${chalk.yellow('Blocking')}: ${ruleEvaluation.is_blocking}\n`
+  if (ruleEvaluation.details_url && ruleEvaluation.details_url.length > 0) {
```

Comment:
is `details_url` always a `string`? If that's the case, this is enough. An empty string is falsy 
```suggestion
  if (ruleEvaluation.details_url) {
```

Reply:
Oh, thanks!